pkg/utils/utils.go

```diff
@@ -165,6 +165,10 @@ func renderDisplayableList(items []Displayable, config RenderListConfig) (string
 
 // RenderTable takes an array of string arrays and returns a table containing the values
 func RenderTable(stringArrays [][]string) (string, error) {
+	// if empty array is given then getPadWidths will panic hence check to make sure array is not empty
+	if len(stringArrays) == 0 {
+		return "", errors.New("RenderTable given an empty array")
```

Comment:
I don't actually think this is worthy of an error. I would instead simply return an empty string and a nil error.

Reply:
Returning an empty string and nil error if `len(stringArrays) == 0`